### PR TITLE
Adds T1 stock parts to assembly vendomats

### PR DESCRIPTION
**Pull request by jakmak3**
This PR adds in some T1 stock parts to the vendomats located in technical storage and tool storage. These are the same vendomats that vend remote signalers and igniters. Also when hacked to unlock contraband these same vendomats can also vend 30u bottles of sulfuric acid. The old PR of this bugged and i had to re do it.

:cl: 
*adds T1 stockparts to the assembly vendomats

### CODE REVIEW

**Comment by PJB3005** [comment]
Still blew it.

Get into discord or IRC so we can help you.

**Comment by FudgePucker** [comment]
:+1:
Now you don't have to loot machines around the station for parts.

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 1]
```diff
@@ -404,3 +404,11 @@
 	New()
 		..()
 		reagents.add_reagent(BICARODYNE, 30)
+
+/obj/item/weapon/reagent_containers/glass/bottle/sacid
+	name = "Sulphuric Acid Bottle"
+	desc = "A small bottle. Contains a small amount of Sulphuric Acid"
```
Comment (PJB3005) [inline code/modules/reagents/reagent_containers/glass/bottle.dm]: Need a period here.

**Review by PJB3005 — APPROVED** [review]

**Comment by Kurfursten** [comment]
Why sulphuric acid? Other than to give greytide instant autolathes roundstart now with the solder I mean. You could add it to the engivend I suppose so it would at least be in engineering, but I think it's overkill

Even then I'm not a huge fan. You can get stock parts from cargo, mechanics, and scientists. That's three options for departments, plus as Fudge says you can loot machines if you're being totally illicit. I know Fudge campaigns long and hard to make people interact less, and I'm probably going to get four paragraphs of all caps text complaining about how scientists are never at the window, but stock parts are not hard to come by legally. I've never had trouble asking at cargo, they're thrilled to get the business.

**Comment by jakmak3** [comment]
I added this in primarily to give the shitty vendomats that basically function as nice aesthetic pieces currently a use and also since box and metaclub technical storage has no stock parts in them and deficiency tech storage has only a few telecoms stock parts. Also i added this in to reduce the amount of greyshittery that breaks into cargo or science or chem to get their 20u of sulph acid or their 4 manipulators and to distract them from ripping apart the stations machinery for parts. Besides the common greyshit doesnt care enough to bother making a autolathe and this would shave 3 minutes off the proccess of making one. I also added in the sulph acid since there is 2 places to get them which is chem and science both of which always have a chance of telling you to fuck off when you ask for anything.

**Comment by FudgePucker** [comment]
@Kurfursten WHAT THE FUCK DID YOU SAY ABOUT ME YOU LITTLE BITCH? I WILL HAVE YOU KNOW I GRADUATED FROM THE NAVY SEALS AT THE TOP OF MY CLASS.
In all honesty you make a decent point. But I think that the vendomats should have stock parts, but move the acid to contraban engineer vends, general access to acid might be a bad idea.
The only machine that you can loot without breaking into a department for a capuator is the solar SMESs. Even then its weakening the power grid and breaking into a secure area.
Also I am not against interaction. I am against dumping features into feature sink departments for the sake of forcing interaction. All the other departments can't have a shiny toy round start? Oh just lock it in the protolathe and make them go to science, that solves it. So the situation becomes where science and cargo gets all the shiny toys and the rest of the station gets jack shit. Cargo and science don't have to leave their departments to do their jobs. Xenobio doesn't need to beg and plead to get something from a department to do their job. Toxins doesn't have to be at the mercy of another players whims to have a productive shift. You expect the other departments to interact to get something from the department that doesn't need to interact.
Also there are people that refuse to go to sleep during surgury, you expect me to be able to get a bottle of acid from this player base?

**Comment by Kurfursten** [comment]
@jakmak3 Acid is the same as the parts, you have two departments to ask or you can make your own with ghettochem. Seems to me "make stuff public so people don't break in" isn't a great justification. Not only are you removing the need to interact with departments to get acid to begin with (making their shift more boring), but you remove the need for security who need something to do during extended. I do feel you about making the assembly machine feel less useless, though. It is overall one of the least enticing machines. I'm not sure stock parts are the direction to go. Maybe storing 50u beakers would make the machine more attractive?

@FudgePucker If you want to make Cargo more reliant on interacting, I would be thrilled to see wrapping paper and pizza crates removed so that they have to go to Botany for the biogenerator/food. It would also be interesting if Engineering had a machine they could divert excess power to which would speed up cargo shuttle travel times dramatically. Kill two birds with one stone, Probe has been asking for a use for extra power for 2 years now.

(Also just a note for your powergaming benefit, you can get capacitors from the cell chargers in tools and pod bay, or from the cyborg charging station in maint, or by ransacking the stacker in recycling)

As for science, there are a lot of things I would personally do to force science to interact more that are stopped because science can build every department. I'm not happy with it, but I can't change it. I can only try to stop other departments from getting as bad as science is. It's a problem I've contributed to in the past too, research botanist was a mistake.

**Comment by Kurfursten** [comment]
I still think this is a bad idea, but I guess it's getting merged anyway against the github votes.

**Comment by clusterfack** [comment]
I had him remove sacid and put it in the engi vend instead as everyone here commented, also don't go counting your emoji votes after the PR is already merged

**Comment by Kurfursten** [comment]
It was 4-3 at the time-of-merge. Not that I'll throw a fit over it, but if you're going to say you're making your choice based on emojis, at least have your numbers right.

I mean, it's one thing to say redditvotes don't really matter because at a certain point they're just an abstraction and not that important. But if the votes don't matter you have to have some rationale for what you merge - so just having that would've been fine too.

**Comment by clusterfack** [comment]
4-3 in favor that is

**Pull request merged**